Sluggo/View Controllers/TeamTableViewController.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  TeamTableViewController.swift
+//  Sluggo
+//
+//  Created by Samuel Schmidt on 5/3/21.
+//
+
+import UIKit
+
+class TeamTableViewController: UITableViewController {
+    var identity: AppIdentity!
+    var completion: ((TeamRecord) -> Void)?
+    private var isFetching: Bool = false
+    private var maxNumber: Int = 0
+    private var teams: [TeamRecord] = []
+    
+    override func viewDidLoad() {
+        self.configureRefreshControl()
+        self.loadData(page: 1)
+    }
+    
+    func configureRefreshControl() {
+        refreshControl = UIRefreshControl()
+        refreshControl?.addTarget(self, action: #selector(handleRefreshAction), for: .valueChanged)
+    }
+    
+    @objc func handleRefreshAction() {
+        teams = []
+        self.loadData(page: 1)
+    }
+    
+    private func preselectRow() {
```

Comment:
Encountered a fatal error here: In my testing this crashes if there are no teams currently on the instance. This is OK if we are going to address this in a future commit, but ideally we could add some error to prevent this code path from executing.

Reply:
I didn't test for that, will resolve